sdks/python/apache_beam/examples/wordcount.py

```diff
@@ -45,17 +40,21 @@ def process(self, context):
     The element is a line of text.  If the line is blank, note that, too.
 
     Args:
-      context: the call-specific context: data and aggregator.
+      context: the call-specific context.
 
     Returns:
       The processed element.
     """
     text_line = context.element.strip()
     if not text_line:
-      context.aggregate_to(empty_line_aggregator, 1)
+      empty_line_counter = Metrics.counter(self.__class__, 'emptyLines')
+      empty_line_counter.inc(1)
     words = re.findall(r'[A-Za-z\']+', text_line)
     for w in words:
-      context.aggregate_to(average_word_size_aggregator, len(w))
+      words_counter = Metrics.counter(self.__class__, 'words')
```

Comment:
Do you need to create a new counter for each word? Can you move this out of the loop?